.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
         name: pylint (library code)
         types: [python]
         args:
-          - --disable=consider-using-f-string
+          - --disable=consider-using-f-string,duplicate-code
```

Comment:
was the duplicate-code disable flag here included intentionally? It looks like this change was not included in #23 for the library code. It is for the example code though, so perhaps the idea is to keep them consistent?

Reply:
Ah, probably not, I think I misunderstood the instructions when working on applying that patch and that was the mistake I was making.  Let me remove it, since it's worked fine in the other branch so it's not needed.